web/src/components/tasks/business-formation/business/BusinessStep.test.tsx

```diff
@@ -489,77 +494,121 @@ describe("Formation - BusinessStep", () => {
       });
     }
 
-    describe("Business Designator Options based on Will Practice Law Answer", () => {
-      it.each(corpLegalStructures)(
-        "Shows PA and PC options for Business Designator when Will You Practice Law is Yes",
-        async (legalStructureId) => {
-          await getPageHelper({ businessPersona: "FOREIGN", legalStructureId }, { willPracticeLaw: true });
+    describe("Business Designator", () => {
+      describe("Business Designator secondary label foreign corporation", () => {
+        it.each(corpLegalStructures)(
+          "Shows secondary label foreign corporation",
```

Comment:
Add the legal structure variable

Reply:
@FarazA22 you mean add it to the name of the test so it's more obvious what is being tested correct?

So it would look something like this?
```bash
# terminal output
c-corp - Shows secondary label
s-corp - Shows secondary label
...
```